lib/routes/account.js

```diff
@@ -65,9 +66,9 @@ module.exports = function (
               name: isA.string().max(255).required(),
               type: isA.string().max(16).required(),
               pushCallback: isA.string().uri({ scheme: 'https' }).max(255).optional().allow(''),
-              // We're not yet ready to store pubkey values, don't let clients submit them.
-              pushPublicKey: isA.string().length(64).regex(HEX_STRING).allow('').forbidden()
-            })
+              pushPublicKey: isA.string().max(88).regex(URLSAFEBASE64).optional().allow(''),
+              pushAuthKey: isA.string().max(24).regex(URLSAFEBASE64).optional().allow('')
```

Comment:
Will these ever be smaller than 88 or 24 bytes?  If we expect them to be exactly that long perhaps we should use `length()` rather than `max()` here.